modules/ROOT/pages/sap/sap-s4-hana-connector-reference.adoc

```diff
@@ -0,0 +1,518 @@
+| Configuration | String | The name of the configuration to use. | | *x*{nbsp}
+| Service a| String |  +++The type of service. This is defined from the services available in the SAP S/4HANA instance.+++ |  | *x*{nbsp}
+| Entity type a| String |  +++The type of entity over which the operation is executed. This is defined from the types of entities available in the selected service.+++ |  | *x*{nbsp}
+| Entity a| Object |  +++The entity to be affected in the SAP S/4HANA instance. This entity must be of the type defined in the type parameter.+++ |  +++#[payload]+++ | {nbsp}
+| Custom headers a| Object |  +++Custom headers that the request will include. The ones specified here will overwrite the default ones specified in the config.+++ |  | {nbsp}
+| Custom query parameters a| Object |  +++Custom query parameters that the request will include. The ones specified here will overwrite the default ones specified in the config.+++ |  | {nbsp}
+| Target Variable a| String |  +++The name of a variable on which the operation's output will be placed+++ |  | {nbsp}
+| Target Value a| String |  +++An expression that will be evaluated against the operation's output and the outcome of that expression will be stored in the target variable+++ |  +++#[payload]+++ | {nbsp}
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors+++ |  | {nbsp}
+|======================
+
+==== Output
+[cols=".^50%,.^50%"]
+|======================
+| *Type* a| Object
+|======================
+
+==== For Configurations.
+* <<config>> {nbsp}
+
+==== Throws
+* S4HANA:TIMEOUT {nbsp}
+* S4HANA:CONNECTIVITY {nbsp}
+* S4HANA:NO_SUCH_SERVICE {nbsp}
+* S4HANA:NO_SUCH_ENTITY_TYPE {nbsp}
+* S4HANA:INVALID_KEY {nbsp}
+* S4HANA:NO_SUCH_ENTITY_KEY {nbsp}
+* S4HANA:RETRY_EXHAUSTED {nbsp}
+* S4HANA:UNAUTHORIZED {nbsp}
+* S4HANA:INVALID_ENTITY {nbsp}
+
+
+[[deleteEntity]]
+=== Delete entity
+`<s4hana:delete-entity>`
+
++++
+Executes a delete operation within the connected SAP S/4HANA instance.
++++
+
+==== Parameters
+[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+|======================
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | *x*{nbsp}
+| Service a| String |  +++The type of service. This is defined from the services available in the SAP S/4HANA instance.+++ |  | *x*{nbsp}
+| Entity type a| String |  +++The type of entity over which the operation is executed. This is defined from the types of entities available in the selected service.+++ |  | *x*{nbsp}
+| Returned Fields a| String |  +++The comma separated list of fields of the entity to return. If no field is selected, then all fields are returned.+++ |  +++*+++ | {nbsp}
+| Entity key a| Object |  +++The object that identifies the entity sought. This may be an object as of itself. This is defined from the type of entity selected.+++ |  +++#[payload]+++ | {nbsp}
```

Comment:
What does "This may be an object as of itself" mean? Does it mean that the entity may be an object? If yes, please change to "This may be an object."